packages/adapters/wagmi/src/client.ts

```diff
@@ -257,6 +257,22 @@ export class WagmiAdapter extends AdapterBlueprint {
       }
     }
 
+
+    if (options.enableSafe !== false) {
```

Comment:
Should we do this different? Should we instead check `window` if the user is on `https://app.safe.global/`?

Because this should only be loaded if the user is there

Reply:
Yes will do that instead cause the connector does not work otherwise (internally `getProvider` checks that it's on an iframe)

Reply:
https://github.com/safe-global/safe-apps-sdk/tree/main/packages/safe-apps-sdk#manifest

We should link to this in the docs if people are intending to use safe as it won't work if not present (adding to lab rn)